HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentView.swift

```diff
@@ -165,19 +165,25 @@ final class PaymentComponentView: UIView {
     
     private func setupView() {
         self.translatesAutoresizingMaskIntoConstraints = false
-        self.frame = CGRect(x: 0, y: 0, width: .max, height: 240)
-        
+        self.frame = CGRect(x: 0, y: 0, width: .max, height: 220)
+
         self.backgroundColor = viewModel.backgroundColor
         
         self.addSubview(contentStackView)
         
         // Content StackView Constraints
+        let contentViewHeightConstraint = heightAnchor.constraint(equalToConstant: frame.height)
+        contentViewHeightConstraint.priority = .required - 1 // We need this to silent warnings
+        contentViewHeightConstraint.isActive = true
+
+        let contentViewBottomAnchorConstraint = contentStackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 4)
```

Comment:
[suggestion]: Can you please move the `constant: 4` in  a `private enum Constants {`?
Also for ` contentStackView.topAnchor.constraint(equalTo: topAnchor, constant: 16),` and for the height of the frame.
Please check all the constants and move them in the enum if possible.

Reply:
This one is fixed in next PRs. 👍